### ISSUE TITLE
Is there a mirroring/symmetrical option?

### ISSUE THREAD

**Issue by elanhickler** [wjakob/instant-meshes#120]
Is there a way to get a symmetrical mesh?

Edit: Do not @ me, learn 3D skills instead.

**Comment by SomeAB** [comment]
my questions as well

**Comment by elanhickler** [comment]
Try dividing your mesh in half and mirroring in another 3d program

**Comment by SomeAB** [comment]
That leaves a fine edge in the middle.  Already tried that @elanhickler

**Comment by AlexDotel** [comment]
Me too, and doesn't work.

> That leaves a fine edge in the middle. Already tried that @elanhickler

**Comment by mabo77** [comment]
yeah this leaves a fine edge in the middle try using zBrush and it wont or clean topology by hand 
cheers m@
